### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by shreyash2106**

### CODE REVIEW

**Comment by cheungdaven** [comment]
Shall we make a website like https://auto.gluon.ai?

**Comment by shreyash2106** [comment]
> Shall we make a website like https://auto.gluon.ai?

I think we should, yeah. But maybe let's wait for the name to go through IP approval before we make a website?

**Review by cheungdaven — APPROVED** [review]
lgtm

**Pull request merged**
